www/app/categories/categories.controller.js

```diff
@@ -33,6 +33,23 @@
         ctrlScope.selectedCategory = category;
       };
 
+      ctrlScope.showConfirmSubscribe = function() {
+        var confirmPopup = $ionicPopup.confirm({
+          title: 'SmartQuote App',
+          template: '¿Desea subscribirse a la Categoria?'
+        });
+        confirmPopup.then(function(res) {
```

Comment:
@rfrancoch, the function has 'res ' as a parameter, which is the response of the pop-up ask